src/python/pants/bin/pants_runner.py

```diff
@@ -27,6 +27,7 @@
 
 # First version with working Python 3.11 support:
 # https://github.com/pantsbuild/scie-pants/releases/tag/v0.12.2
+# TODO: Likely still need to update scie-pants
```

Comment:
NOTE TO SELF... Need scie-pants update before this lands (though, I haven't actually tested scie-pants on this, I just don't think we have 3.14 support in there yet.

Reply:
Is there a tracking issue for proving out required scie-pants updates?